src/fswAlgorithms/attControl/rateDamp/rateDamp.h

```diff
@@ -17,16 +17,15 @@
 
  */
 
-#ifndef _RATE_DAMP_
-#define _RATE_DAMP_
+#ifndef BASILISK_RATE_DAMP_H
+#define BASILISK_RATE_DAMP_H
 
 #include "architecture/_GeneralModuleFiles/sys_model.h"
 #include "architecture/messaging/messaging.h"
-#include "architecture/msgPayloadDefC/CmdForceBodyMsgPayload.h"
 #include "architecture/msgPayloadDefC/CmdTorqueBodyMsgPayload.h"
 #include "architecture/msgPayloadDefC/NavAttMsgPayload.h"
-#include "architecture/utilities/avsEigenSupport.h"
 #include "architecture/utilities/bskLogging.h"
+#include "fswAlgorithms/attControl/rateDamp/rateDampAlgorithm.h"
 
 /*! @brief A class to compute rate damping control */
```

Comment:
:small_red_triangle_down: I know it's not part of this PR, but can you add `override` to the methods reset and updateState?